manifests/master/pdf_stack.pp

```diff
@@ -1,7 +1,7 @@
 # Dependencies for PDF rendering
 class classroom::master::pdf_stack {
 
-  $enabled = $classroom::offline ? {
+  $enabled = defined('$classroom::offline') and $classroom::offline ? {
```

Comment:
@samuelson Thought about that. I would need to add it to this class and the calling class `classroom::master::showoff`. I had to do something similar with the new `jvm_tuning_profile` parameter.